subschedulerevision.go

```diff
@@ -0,0 +1,57 @@
+package stripe
+
+import (
+	"encoding/json"
+)
+
+// SubscriptionScheduleRevisionParams is used when retrieving a schedule revision.
+type SubscriptionScheduleRevisionParams struct {
+	Params   `form:"*"`
+	Schedule *string `form:"-"` // Included in the URL
+}
+
+// SubscriptionScheduleRevisionListParams is the set of parameters that can be used when listing
+// subscription schedules.
+type SubscriptionScheduleRevisionListParams struct {
+	ListParams `form:"*"`
+	Schedule   *string `form:"-"` // Included in the URL
+}
+
+// SubscriptionScheduleRevision is the resource representing a Stripe subscription schedule revision.
+type SubscriptionScheduleRevision struct {
+	Created          int64                                `json:"created"`
+	ID               string                               `json:"id"`
+	Livemode         bool                                 `json:"livemode"`
+	Metadata         map[string]string                    `json:"metadata"`
+	Object           string                               `json:"object"`
+	Phases           []*SubscriptionSchedulePhase         `json:"phases"`
+	PreviousRevision *SubscriptionSchedule                `json:"previous_revision"`
```

Comment:
Is this expandable? The API ref docs are showing that it's just a string.

Reply:
@brandur-stripe It's not expandable but if it became expandable it'd be a breaking change so I preferred to do this. It might be the wrong approach though. What do you think is better?

Reply:
Yeah, that works for me. Just wanted to make sure you did it on purpose.

Reply:
Argl yeah I remember they added this after I started the implementation. Fixing now.